stacks-common/src/util/vrf.rs

```diff
@@ -432,7 +429,7 @@ impl VRF {
     /// * its public key (an ed25519 curve point)
     /// * a new private key derived from the hash of the private key
     /// * a truncated hash of the private key
-    /// Idea borroed from Algorand (https://github.com/algorand/libsodium/blob/draft-irtf-cfrg-vrf-03/src/libsodium/crypto_vrf/ietfdraft03/prove.c)
+    ///   Idea borroed from Algorand (https://github.com/algorand/libsodium/blob/draft-irtf-cfrg-vrf-03/src/libsodium/crypto_vrf/ietfdraft03/prove.c)
```

Comment:
Typo (that already existed): `borroed` -> `borrowed`. Also there should be a newline, the "idea borrowed from algorand" is referring to the whole function (I believe)

Reply:
This space will give it a new paragraph. Do we want it as a new line and NOT paragraph?